src/cpp/README.md

```diff
@@ -1,17 +1,12 @@
-
-# Overview
-
-A C++ implementation of gRPC
-
-# To start using gRPC C++
+# Add gRPC as a dependency to your project
```

Comment:
nit: keep the "To start using gRPC C++" header and make first sentence "This section describes how to add gRPC as a dependency to your C++ project".

Reply:
👍